templates/_base.html

```diff
@@ -10,6 +10,19 @@
     <meta name="generator" content="cot-site based on Cot framework">
     <title>{% block title %}{% endblock %} | Cot</title>
 
+    <!-- Open Graph Meta Tags -->
+    <meta property="og:title" content="Cot">
+    <meta property="og:description" content="The Rust web framework for lazy developers. Build secure, type-safe web apps with ease.">
+    <meta property="og:image" content="https://cot.rs/static/images/favicon-512.png">
+    <meta property="og:url" content="https://cot.rs">
+    <meta property="og:type" content="website">
+
+    <!-- Twitter/X Card Meta Tags -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:title" content="Cot">
+    <meta name="twitter:description" content="The Rust web framework for lazy developers. Build secure, type-safe web apps with ease.">
+    <meta name="twitter:image" content="https://cot.rs/static/images/favicon-512.png">
```

Comment:
Twitter supports using OpenGraph meta tags values, so no need to duplicate: https://developer.x.com/en/docs/x-for-websites/cards/overview/markup
```suggestion
```